CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## 0.0.13
+Released on September 6, 2022.
+
+### Key Features
+* Adds AWS Athena integration. You can now execute SQL queries against AWS Athena using the Aqueduct integration API. 
+    (Since Athena is a query service, we do not support saving data to Athena.)
+
+### Enhancements
+* Simplifies the presentation of the notifications pane to be a single box.
```

Comment:
Maybe we change this to "Removes team and workflow notification categories and simplifies presentation of notifications pane to be a single box containing all notifications."